package.json

```diff
@@ -44,7 +44,7 @@
                 {
                     "id": "json-outline",
                     "name": "JSON Outline",
```

Comment:
nit: Since these only apply to ARM - can ARM be in the title of the explorer? "JSON Outline" seems too generic
![screen shot 2018-02-01 at 10 19 16 am](https://user-images.githubusercontent.com/11282622/35695475-71f522b0-0739-11e8-9c7f-6b9527f9ded1.png)